src/Api/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -38,7 +39,7 @@ public static void AddSwagger(this IServiceCollection services, GlobalSettings g
                         TokenUrl = new Uri($"{globalSettings.BaseServiceUri.Identity}/connect/token"),
                         Scopes = new Dictionary<string, string>
                         {
-                            { "api.organization", "Organization APIs" },
+                            { Scopes.ApiOrganization, "Organization APIs" },
```

Comment:
I feel moving the magic string "Organization APIs" to the Utilities too will give us a uniform place to make change